business/services.go

```diff
@@ -118,6 +182,20 @@ func (in *SvcService) GetService(namespace, service, interval string, queryTime
 	var vs, dr []kubernetes.IstioObject
 	var ws models.Workloads
 	var nsmtls models.MTLSStatus
+	var apidoc models.ApiDocumentation
+
+	conf := config.Get()
+	apiSpecFromAnnotation := svc.ObjectMeta.Annotations[conf.ApiDocumentation.Annotations.ApiSpecAnnotationName]
+	apiTypeFromAnnotation := svc.ObjectMeta.Annotations[conf.ApiDocumentation.Annotations.ApiTypeAnnotationName]
+	apiBaseUrl := ""
+	if apiSpecFromAnnotation != "" {
+		apiBaseUrl = conf.Server.WebRoot + "/api/namespaces/" + namespace + "/services/" + service
```

Comment:
It doesn't work for me: my WebRoot is `/` so it's ending up in something like: `//api/namespaces/istio-system/services/istio-pilot/apispec` , which fails & I can't see the petstore doc that i've configured in annotations.

Actually I think this `apiBaseUrl` is not needed at all because we can build it from the frontend, just like we build any other API url. I'll propose a patch to your frontend PR about that.

Reply:
@couniojc can you rebase your 2 PRs from master? You'd have to do that anyway, and it's also needed to fix an issue on the frontend to fix the build.

Reply:
@jotak Thanks, ok I will rebase, and yes I had back and forth about this baseUrl, but you're right, it will be cleaner if it is built in the frontend, let me see if I can change it